addons/binding/org.openhab.binding.hp1000/src/main/java/org/openhab/binding/hp1000/handler/UnitConverterUtils.java

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
As mentioned before you can remove this class if you make use of our new units of measurement feature :)